icepyx/core/validate_inputs.py

```diff
@@ -104,3 +104,16 @@ def tracks(track):
             warnings.warn("Listed Reference Ground Track is not available")
 
         return track_list
+
+def check_s3bucket(path):
+    """
+    Check if the given path is an s3 path. Raise a warning if the data being referenced is not
+    in the NSIDC bucket
+    """
+    split_path = path.split('/')
+    if split_path[0] == 's3:' and split_path[2] != 'nsidc-cumulus-prod-protected':
+        warnings.warn(
+            's3 data being read from outside the NSIDC data bucket. Icepyx can '
+            'read this data, but available data lists may not be accurate.', stacklevel=2
+        )
+    return path
```

Comment:
Should this go in the warnings module since it's only output is warning the user if they're not in the NSIDC bucket? The path itself is not modified (so does it need to be returned?).

Reply:
I see how you're using this in `Variables` and it all makes sense.

Reply:
Either warnings or validate inputs seems logical to me. I'll trust your judgement on this one; let me know if you'd like it moved.